2007-11-19  Ulrich Drepper  <drepper@redhat.com>

	* posix/gai.conf: Document scopev4 defaults.

2007-11-19  Ulrich Drepper  <drepper@redhat.com>

	* posix/gai.conf: Update for current default tables.

===================================================================
RCS file: /cvs/glibc/libc/posix/gai.conf,v
retrieving revision 1.2
retrieving revision 1.3
--- glibc-2.7.orig/posix/gai.conf
+++ glibc-2.7/posix/gai.conf
@@ -19,13 +19,14 @@
 #    Add another rule to the RFC 3484 label table.  See section 2.1 in
 #    RFC 3484.  The default is:
 #
-#label  ::1/128       0
-#label  ::/0          1
-#label  2002::/16     2
-#label ::/96          3
-#label ::ffff:0:0/96  4
-#label  fec0::/10     5
-#label  fc00::/7      6
+#label ::1/128       0
+#label ::/0          1
+#label 2002::/16     2
+#label ::/96         3
+#label ::ffff:0:0/96 4
+#label fec0::/10     5
+#label fc00::/7      6
+#label 2001:0::/32   7
 #
 #    This default differs from the tables given in RFC 3484 by handling
 #    (now obsolete) site-local IPv6 addresses and Unique Local Addresses.
@@ -35,10 +36,11 @@
 #    site-local IPv4 and IPv6 addresses a lookup for a global address would
 #    see the IPv6 be preferred.  The result is a long delay because the
 #    site-local IPv6 addresses cannot be used while the IPv4 address is
-#    (at least for the foreseeable future) NATed.
+#    (at least for the foreseeable future) NATed.  We also treat Teredo
+#    tunnels special.
 #
 # precedence  <mask>   <value>
-#    Add another rule the to RFC 3484 precedence table.  See section 2.1
+#    Add another rule to the RFC 3484 precedence table.  See section 2.1
 #    and 10.3 in RFC 3484.  The default is:
 #
 #precedence  ::1/128       50
@@ -50,3 +52,17 @@
 #    For sites which prefer IPv4 connections change the last line to
 #
 #precedence ::ffff:0:0/96  100
+
+#
+# scopev4  <mask>  <value>
+#    Add another rule to the RFC 3484 scope table for IPv4 addresses.
+#    By default the scope IDs described in section 3.2 in RFC 3484 are
+#    used.  Changing these defaults should hardly ever be necessary.
+#    The defaults are equivalent to:
+#
+#scopev4 ::ffff:169.254.0.0/112  2
+#scopev4 ::ffff:127.0.0.0/104    2
+#scopev4 ::ffff:10.0.0.0/104     5
+#scopev4 ::ffff:172.16.0.0/108   5
+#scopev4 ::ffff:192.168.0.0/112  5
+#scopev4 ::ffff:0.0.0.0          14
